crates/bevy_core_pipeline/src/core_2d/mod.rs

```diff
@@ -19,6 +19,8 @@ pub mod graph {
         MainOpaquePass,
         MainTransparentPass,
         EndMainPass,
+        StartDebugPass,
```

Comment:
I don't love the debug naming for this, personally.

Reply:
Happy for alternate suggestions.

Reply:
Can we just name it WireframePass?